libs/SalesforceReact/SalesforceReactTests/ReactSmartSyncTests.m

```diff
@@ -23,9 +22,44 @@
  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#import <UIKit/UIKit.h>
+#import <SmartStore/SmartStore.h>
+#import <SmartSync/SmartSync.h>
+#import "ReactTestCase.h"
 
-#import "SFPluginTestSuite.h"
+@interface ReactSmartSyncTests : ReactTestCase
```

Comment:
Tests for the smartsync react bridge.
Pretty much calling every method we expose.